entity_event/models.py

```diff
@@ -1066,3 +1058,142 @@ def _unseen_event_ids(medium):
     unseen_events = Event.objects.raw(query, params=[medium.id])
     ids = [e.id for e in unseen_events]
     return ids
+
+
+@python_2_unicode_compatible
+class RenderingStyle(models.Model):
+    """
+    Defines a rendering style. This is used to group together mediums that have
+    similar rendering styles and allows context renderers to be used across
+    mediums.
+    """
+    name = models.CharField(max_length=64, unique=True)
+    display_name = models.CharField(max_length=64, default='')
+
+    def __str__(self):
+        return self.display_name
+
+
+@python_2_unicode_compatible
+class ContextRenderer(models.Model):
+    """``ContextRenderer`` objects store information about how
+    a source or source group is rendered with a particular rendering style, along with
+    information for loading the render context in a database-efficient
+    manner.
+
+    Of the four template fields: `text_template_path`, 'html_template_path',
+    `text_template`, and `html_template`, at least one must be
+    non-empty. Both a text and html template may be provided, either
+    through a path to the template, or a raw template object.
+    However, for either text or html templates, both a path and raw
+    template should not be provided.
```

Comment:
Will update that comment, although these docs are directly copied from another repo :)